regex-parsing/src/main/java/org/sonarsource/analyzer/commons/regex/finders/EmptyAlternativeFinder.java

```diff
@@ -31,24 +35,66 @@
 
 public class EmptyAlternativeFinder extends RegexBaseVisitor {
 
-  private static final String MESSAGE = "Remove this empty alternative.";
+  private static final String MESSAGE_REMOVE_THIS = "Remove this empty alternative.";
```

Comment:
Why did you change the constant name? MESSAGE is clear in almost all check aka finder contexts. Something which is pre or postfix with "REMOVE_THIS" looks more like a leftover or ugly TODO. I've got your idea, but I really don't like it.

Reply:
To give you an idea how it got here:
That's a leftover from the brief time I had couple of different messages MSG_REMOVE_THIS & MSG_REMOVE_EITHER